claim-169/Readme.md

```diff
@@ -0,0 +1,139 @@
+| 6         | tstr  | Middle Name          | Middle name of the citizen                                                                                                                                                                                                      |
+| 7         | tstr  | Last Name            | Last name of the citizen                                                                                                                                                                                                        |
+| 8         | tstr  | Date of Birth        | Date of birth of the citizen                                                                                                                                                                                                    |
+| 9         | tstr  | Gender               | Gender of the citizen                                                                                                                                                                                                           |
+| 10        | tstr  | Address              | Address of the citizen with address line separator character `\n`                                                                                                                                                               |
+| 11        | tstr  | Email ID             | Email id of the citizen                                                                                                                                                                                                         |
+| 12        | tstr  | Phone Number         | Contact number of the citizen                                                                                                                                                                                                   |
+| 13        | tstr  | Nationality          | Nationality of the citizen                                                                                                                                                                                                      |
+| 14        | tstr  | Marital Status       | Marital status of the citizen                                                                                                                                                                                                   |
+| 15        | tstr  | Guardian             | Name of the entity playing the role of a guardian of the citizen such as mother, father, spouse, sister, legal guardian etc.                                                                                                    |
+| 16        | tstr  | Binary Image         | Binary image data representing the citizen's photograph                                                                                                                                                                         |
```

Comment:
Binary image format? We will need a separate attribute and it can contain the following values 1 - means jpeg, 2 - Jpeg2, 3 - AVIF, 4- Webp,

Reply:
Added new attribute at 17 and description. Also updated the Sample data.